about.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <title>Prog Rock Revival Concert</title>
+</head>
+<body>
+  <header>
+		<a class="header-icon" href="#">
+			<i class="fa-brands fa-facebook-f"></i>
+		</a>
+    <a class="header-icon" href="#">
+			<i class="fa-brands fa-instagram"></i>
+		</a>
+    <a class="header-icon" href="#">
+			<i class="fa-brands fa-twitter"></i>
+		</a>
+    <a class="header-icon" href="#">
+			<p>English</p>
+		</a>
+  </header>
+  <nav>
+    <a class="hamb-icons" id="menuHamburger">
+			<i id="hamburgerIcon" class="fa-solid fa-bars"></i>
+			<i id="closeIcon" class="fa-solid fa-xmark"></i>
+		</a>
+    <div class="prr-logo">
+      <a href="index.html">
+        <img src="./imgs/prog-rock-rev-logo.png" alt="prog-rock-revival logo">
+      </a>
+    </div>
+    <div class="nav-items" id="items-menul">
+      <ul class="ul-menu">
+        <li>
+          <a href="about.html" id="about-link">About</a>
```

Comment:
- [ ] I kindly suggest that you change this `<a href>` to link to home page instead of linking back to itself.

## EXAMPLE
`<li><a href="./index.html">Home</a></li>`